source/Kaboom.cs

```diff
@@ -28,23 +28,21 @@ public class ModuleKaboom : PartModule
         [KSPField(isPersistant = true)]
         public bool isGlued = false;
 
-        [KSPField(guiName = "Superglue", guiActive = true, guiActiveEditor = true, groupName = "Kaboom")]
+        [KSPField(isPersistant = true, guiName = "Superglue", guiActive = true, guiActiveEditor = true, groupName = "Kaboom")]
```

Comment:
@yalov "Superglue it" love it - keeps up the kampy/cheeky/not taking itself too seriously attitude of KaboOom!